public/app/features/explore/utils/links.ts

```diff
@@ -60,3 +61,23 @@ function getTitleFromHref(href: string): string {
   }
   return title;
 }
+
+function buildHostUrl() {
```

Comment:
This is duplicated from the dashboard solution. I plan to unify both solutions and move this links-utility functions, so both - dashboard and explore - can use them. 
I have added it to the list: https://github.com/grafana/grafana/issues/28248